apps/studio/pages/new/[slug].tsx

```diff
@@ -146,6 +146,11 @@ const Wizard: NextPageWithLayout = () => {
     ''
   )
 
+  // This is to make the database.new redirect work correctly. The database.new redirect should be set to supabase.com/dashboard/new/last-visited-org
+  if (slug === 'last-visited-org') {
+    router.replace(`/new/${lastVisitedOrganization}`, undefined, { shallow: true })
+  }
```

Comment:
should also consider if `lastVisitedOrganization` is undefined here

```suggestion
  if (slug === 'last-visited-org' && !!lastVisitedOrganization) {
    router.replace(`/new/${lastVisitedOrganization}`, undefined, { shallow: true })
  }
```

Reply:
Fixed both cases, ty for catching!